packages/mobile/e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -1,16 +1,24 @@
-import { enterPinUi, sleep, waitForElementId } from '../utils/utils'
-import { SAMPLE_BACKUP_KEY } from '../utils/consts'
-
-const EXAMPLE_NAME = 'Test Name'
+import { enterPinUi, waitForElementId, sleep } from '../utils/utils'
+import { SAMPLE_BACKUP_KEY, EXAMPLE_NAME } from '../utils/consts'
+import { dismissBanners } from '../utils/banners'
 
 export default RestoreAccountOnboarding = () => {
+  beforeAll(async () => {
```

Comment:
make it into a util function? I think we need it in multiple places.

Reply:
There is [a util function for restore account onboarding](https://github.com/celo-org/wallet/blob/3138944668ff8699aa99bcd288dde5c2dc307671/packages/mobile/e2e/src/utils/utils.js#L136); however, I keep this restore flow to retain individual test specs the restore flow.